### ISSUE TITLE
OMR classification Summer 2019 overview

### ISSUE THREAD

**Issue by vigliensoni** [DDMAL/omr-position-classification#1]
Heuristic approach:

- [x] Create valid MEI 4.0 for Salzinnes ground-truth data
- [ ] Evaluate the performance of the three types of pitch classification (i.e., neume, neume-component, and splicing-based) with MEI ground truth.
- [ ] Evaluate the performance of heuristic pitch finding approaches using Volpiano data

ML-based approach:

- [x] Review paper from Alicia Nuñez-Alcover about Glyph and position classification of music symbols
- [ ] Design a way to generate training set (i.e., getting a set of glyphs with different positions in the staff). This could be done with IC, but the position labels should be retained "per page" (so that we can create a large dataset of positions). Then 
- [ ] Adapt code from paper (https://github.com/HISPAMUS/symbol-classification) to our needs (e.g., variable window size, dependent on some reference length such as staffline height)
- [ ] Implement code job as a Rodan job
